content/src/main/content/jcr_root/home/users/.content.xml

```diff
@@ -16,4 +16,12 @@
   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~-->
 <jcr:root xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:rep="internal"
     jcr:mixinTypes="[rep:AccessControllable]"
-    jcr:primaryType="rep:AuthorizableFolder"/>
+          jcr:primaryType="rep:AuthorizableFolder">
+    <rep:policy
+            jcr:primaryType="rep:ACL">
+        <allowDWREReplicationService
+                jcr:primaryType="rep:GrantACE"
+                rep:principalName="dwre-replication-service"
+                rep:privileges="{Name}[jcr:all]"/>
```

Comment:
this gives the replication system-account full admin rights on the users! why?

Reply:
my poor implementation. 'dwre-replication-service' used to create oauth subnode under instance-specific-user node, so I granted access. I will rewrite AccessTokenProviderImpl and impersonate in specific user.